pkg/lagoon/problems.go

```diff
@@ -0,0 +1,141 @@
+package lagoon
```

Comment:
Hmmm — this is a difficult choice, I like how we have the output package. It clearly illustrates what is an output plugin. However, problems being specific to Lagoon pose a bit of an issue with the convention.

We can leave it here for now, but I'm wondering if we want to make it part of the output package for consistency.

Reply:
It might make more sense in the future when actually implementing the plugin system - see the sample here: https://github.com/salsadigitalauorg/shipshape/pull/69